src/main/java/org/lesson5ex7/Day.java

```diff
@@ -0,0 +1,33 @@
+package org.lesson5ex7;
+
+public class Day {
```

Comment:
w zadaniu miałaś podane że masz stworzyć typ wyliczeniowy czyli enum, a ty stworzyłaś klasę - to bardzo mocno zmienia koncepcje. Przyjrzyj się dokładnie zaganieniu ENUM w lekcji 4 + obejrzyj ponownie wideo w którym opowiadam o tym.